README.md

```diff
@@ -27,11 +27,14 @@ whatever file you use to configure your shell:
 
 ```bash
 vman() {
-  vim -c "SuperMan $*"
+  man -d $* 2> /dev/null > /dev/null
```

Comment:
This is the same as `man -d $* &> /dev/null`. I prefer this form, would you mind changing it?

You can read more about this in `bash(1)` under "Redirecting Standard Output and Standard Error".